modules/hostap/src/supp_api.c

```diff
@@ -24,6 +24,9 @@
 extern struct k_sem wpa_supplicant_ready_sem;
 extern struct wpa_global *global;
 
+/* save the last wifi connection parameters */
+static struct wifi_connect_req_params last_wifi_conn_params;
```

Comment:
can you please add to the commit log, how/where is this used?

Reply:
it's used in NXP wifi code, user can decide to get the recently used wifi connection parameters

Reply:
it would be better to add this to Wi-Fi shell so that we can test/see the usage of the API.

Reply:
Hi @krish2718 , I carefully think about this, and I find it's difficult to add this to wifi shell, as structure of wifi_connect_req_params has too many paramters ( and continue growing), and difficult to parse and print all of them.
if must add the caller, can I only add in wifi_mgmt.c, not the wifi_shell.c?